tests/sentry/models/test_organizationmember.py

```diff
@@ -135,6 +141,22 @@ def test_delete_expired_clear(self):
         OrganizationMember.objects.delete_expired(timezone.now())
         assert OrganizationMember.objects.filter(id=member.id).first() is None
 
+    def test_delete_expired_SCIM_enabled(self):
+        organization = self.create_organization()
+        AuthProvider.objects.create(
+            provider="saml2", organization=organization, flags=AuthProvider.flags["scim_enabled"]
+        )
+        ninety_one_days = timezone.now() - timedelta(days=1)
```

Comment:
This isn't 91 days 🤔

Reply:
o lmao, i copied it from the test case above. I'll change it to 91 days but it seems to not matter since we just set the expiry to the same time